src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -142,9 +142,7 @@ def test_telemetry_pool_metrics(self):
         try:
             self.update_ior_cmd_with_pool(False)
             self.ior_cmd.dfs_oclass.update(self.dfs_oclass)
-            self.ior_cmd.transfer_size.update(self.ior_transfer_size)
-            self.ior_cmd.dfs_chunk.update(self.ior_transfer_size)
-            self.ior_cmd.block_size.update(self.ior_block_size)
+            self.ior_cmd.dfs_chunk.update(self.ior_cmd.transfer_size.value)
```

Comment:
You could also set `dfs_chunk` in the config similar to `transfer_size`: https://github.com/daos-stack/daos/blob/88083a40c67de0ac31336fbc03997979fed23412/src/tests/ftest/telemetry/telemetry_pool_metrics.yaml#L35

Reply:
In fact, I want to be sure that `transfer_size` and `dfs_chunk` have the same size.